documentation/modules/ROOT/pages/index.adoc

```diff
@@ -68,11 +62,11 @@ xref:esmf-developer-guide:tooling-guide:java-aspect-tooling.adoc[Java SDK]
 [.icon-cli]
 --
 [.title]
-Python ESMF SDK
+Python SDK
 
 [.text]
-The Python SDK offers functionality which helps software developers to work with Aspect Models in
-their Python applications.
+Enables software developers to work with Aspect Models in their Python applications.
```

Comment:
Create Aspect APIs in Python for software solutions and analytics use cases.

Reply:
Done with https://github.com/bci-oss/eclipse-esmf.github.io/pull/4/commits/a64917a7941c0b56c98da5dc1af9454db0b0d93c ✔